newscoop/install/Resources/sql/upgrade/4.4.x/2015.09.28/tables.sql

```diff
@@ -1 +0,0 @@
-ALTER TABLE  `topic_translations` ADD  `isDefault` INT( 1 ) NULL DEFAULT NULL;
```

Comment:
Undo the deletion of this file and update the folder name to today. You can leave the changes in newscoop/install/Resources/sql/upgrade/4.4.x/2015.03.11/topics.php .